ribs/emitters/_evolution_strategy_emitter.py

```diff
@@ -0,0 +1,187 @@
+"""Provides the EvolutionStrategyEmitter."""
+import itertools
+
+import numpy as np
+
+from ribs.emitters._emitter_base import EmitterBase
+from ribs.emitters.rankers import RankerBase
+
+
+def get_ranker(ranker_name):
+    try:
+        ranker = globals()[ranker_name]()
+        if issubclass(ranker, RankerBase):
```

Comment:
Why issubclass instead of isinstance?

Reply:
I believe that isinstance will output false if you input (subclass, baseclass)